tests/SerializerTest.php

```diff
@@ -401,6 +401,19 @@ function () {
     expect($f(new Model))->toBeInstanceOf(Model::class);
 })->with('serializers');
 
+test('serializes carbon objects', function () {
+    $carbon = new \Carbon\Carbon('now');
```

Comment:
I think we need a test with carbon immutable, too, and also non-carbon objects

Reply:
I added a test for CarbonImmutable, but I can't replicate the failure with non-Carbon objects. Would you like me to add tests anyways?